src/coreclr/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -7586,6 +7586,39 @@ HRESULT ProfToEEInterfaceImpl::GetObjectIDFromHandle(
     return S_OK;
 }
 
```

Comment:
You'll also need to add `ICorProfilerInfo14` to `ProfToEEInterfaceImpl::QueryInterface`

Reply:
Added